### PR TITLE
Chronos : add regression functions for forecast metrics

### PR DESCRIPTION
**Pull request by Chaselzxy**
## Description

Rewrites the regression functions whose input and output are Numpy. 
### 1. Why the change?

To facilitate customers to use metrics without torchmetrics. Only Numpy is required to complete relevant operations.

### 3. Summary of the change 
Rewrites the regression functions whose input and output are Numpy according to the calculation formula of relevant regression functions.

### 4. How to test?
- [x] Unit test

http://10.112.231.51:18889/job/BigDL-PRNV-chronos-Python-Spark-2.4-py37-Horovod/814/
http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation/784/

### CODE REVIEW

**Comment by TheaperDeng** [comment]
@liangs6212 please also have a look

File: python/chronos/src/bigdl/chronos/metric/forecast_metrics.py [hunk 1]
```diff
@@ -109,26 +186,15 @@ def evaluate(metrics, y_true, y_pred, aggregate='mean'):
...
-                metric_func = TORCHMETRICS_REGRESSION_MAP[metric]
             if len(original_shape) in [2, 3] and aggregate is None:
-                res = torch.zeros(y_true.shape[-1])
+                res = np.zeros(y_true.shape[-1])
                 for i in range(y_true.shape[-1]):
-                    if callable(metric):
-                        res[i] = torch.from_numpy(metric_func(y_true[..., i], y_pred[..., i]))
-                    else:
-                        res[i] = metric_func(y_pred[..., i], y_true[..., i])
+                    res[i] = eval(metric)(y_true[..., i], y_pred[..., i])
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/metric/forecast_metrics.py:192]: we should never use `eval` function since it might be dangerous and vulnerable. just make `REGRESSION_MAP` a dict e.g. `{'mae': mae, 'mse': mse, ...}`

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

File: python/chronos/src/bigdl/chronos/metric/forecast_metrics.py [hunk 2]
```diff
@@ -14,36 +14,123 @@
...
+        \\text{MAE} = \\frac{1}{n}\\sum_{t=1}^n |y_t-\\hat{y_t}|
+    :param y_label: Array-like of shape = (n_samples, \*).
+           Ground truth (correct) target values.
+    :param y_predict: Array-like of shape = (n_samples, \*).
+           Estimated target values.
+    :return: Ndarray of floats.
+             An array of non-negative floating point values (the best value is 0.0).
+    """
+    y_label = np.array(y_label)
+    y_predict = np.array(y_predict)
```
Comment (liangs6212) [inline python/chronos/src/bigdl/chronos/metric/forecast_metrics.py:39]: We can delete all np.array, because ln89 will check the data type.
If you need to convert the input data type, implement it with a function.

**Review by TheaperDeng — APPROVED** [review]
LGTM, pls run jenkins
